src/lro/src/details.rs

```diff
@@ -24,8 +24,8 @@ pub(crate) fn handle_start<R, M>(
     result: Result<Operation<R, M>>,
 ) -> (Option<String>, PollingResult<R, M>)
 where
-    R: wkt::message::Message + serde::de::DeserializeOwned,
-    M: wkt::message::Message + serde::de::DeserializeOwned,
+    R: wkt::message::Message + serde::ser::Serialize + serde::de::DeserializeOwned + 'static,
+    M: wkt::message::Message + serde::ser::Serialize + serde::de::DeserializeOwned + 'static,
```

Comment:
optional: we could avoid these other changes if we separated the serialization and deserialization into two separate traits: `MessageSerializer` and `MessageDeserializer`.

don't feel obligated to do this.

Reply:
For now: The only change I've made here is to remove the static lifetime bound, now that we've reverted back to static dispatch for our `fn serializer()` in the `Message` trait. We can always raise another PR if we think separating `MessageSerializer` would be _more_ beneficial.